go.mod

```diff
@@ -48,6 +48,7 @@ require (
 	github.com/markbates/pkger v0.15.1
 	github.com/onsi/ginkgo v1.12.0 // indirect
 	github.com/onsi/gomega v1.9.0 // indirect
+	github.com/patrickmn/go-cache v2.1.0+incompatible
```

Comment:
Hmm.. No changes to `go.sum`? Did you run `go mod tidy`?

Reply:
yeah, no changes:

```
nikola@avokado ~/src/giantswarm/azure-operator 2020-06-10 10:34:17
$ go mod tidy
nikola@avokado ~/src/giantswarm/azure-operator 2020-06-10 10:34:18
$ git status
On branch azure-clients-factory
Your branch is up to date with 'origin/azure-clients-factory'.

nothing to commit, working tree clean
```